File: app/src/main/java/me/study/silang/ui/main/bbs/PostModel.java
package me.study.silang.ui.main.bbs;

import lombok.*;
import lombok.experimental.Accessors;

import java.io.Serializable;

/**
 * <p>
 * 
 * </p>
 *
 * @author Me
 * @since 2019-03-24
 */
@Data
@EqualsAndHashCode(callSuper = false)
@Accessors(chain = true)
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PostModel implements Serializable {

    private static final long serialVersionUID = 1L;

    private Integer id;

    private String title;

    private String content;

    private String gmtCreate;

    private String gmtUpdate;

    private String userName;

    private Integer replyNum;
}

File: app/src/main/java/me/study/silang/ui/main/bbs/PostModel.kt
package me.study.silang.ui.main.bbs

import lombok.*
import lombok.experimental.Accessors

import java.io.Serializable

/**
 *
 *
 *
 *
 *
 * @author Me
 * @since 2019-03-24
 */
class PostModel : Serializable {

    var id: Int? = null

    var title: String? = null

    var content: String? = null

    var gmtCreate: String? = null

    var gmtUpdate: String? = null

    var userName: String? = null

    var replyNum: Int? = null

    companion object {

        private const val serialVersionUID = 1L
    }
}
